service/geo/app/models/graphql/GraphQlSchemaDefinition.scala

```diff
@@ -23,6 +23,7 @@ object GraphQlSchemaDefinition {
   // Scalar argument types
   val LimitArgument = Argument("limit", IntType, description = "Limit")
   val OffsetArgument = Argument("offset", IntType, description = "Offset")
+  val featureUriArgument = Argument("featureUri", UriType, description= "featureUri")
```

Comment:
Just call this UriArgument, with "uri". We'll need it later.

You notice most of my code is alphabetically ordered within sections. Please try to preserve that.

Reply:
I'd also suggest you look at the Save Actions IntelliJ plugin, which will do things like code formatting on save.